shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
this._sse = new EventSource(this._buildUrl() + "?" + this._buildQuery({
        limitToLast: 1,

But I see below that a GET request is also being made for the last 24 hours of chat.